package.json

```diff
@@ -34,7 +34,7 @@
     "chart.js": "2.9.4",
     "chartjs-plugin-datalabels": "0.7.0",
     "core-js": "3.9.1",
-    "css-loader": "3.6.0",
+    "css-loader": "^5.1.3",
```

Comment:
When rebasing please fix the version here and drop the ^. Using ^ means its harder for developers to reproduce the local environment, it also opens you up to security vulnerabilities by pulling down code from external places that has not been audited by us.